v21.2/alter-role.md

```diff
@@ -45,18 +45,19 @@ Parameter | Description
+`LOGIN`/`NOLOGIN` | Allow or disallow a role to log in with one of the [client authentication methods](authentication.html#client-authentication). Setting the role option to `NOLOGIN` prevents the role from logging in using any authentication method.
 `MODIFYCLUSTERSETTING`/`NOMODIFYCLUSTERSETTING` | Allow or disallow the role to modify the [cluster settings](cluster-settings.html) with the `sql.defaults` prefix. <br><br>By default, the role option is set to `NOMODIFYCLUSTERSETTING` for all non-admin roles.
-`PASSWORD password`/`PASSWORD NULL` | Let the role [authenticate their access to a secure cluster](authentication.html#client-authentication) using this new password. Passwords should be entered as a [string literal](sql-constants.html#string-literals). For compatibility with PostgreSQL, a password can also be entered as an identifier. <br><br>To prevent a role from using [password authentication](authentication.html#client-authentication) and to mandate [certificate-based client authentication](authentication.html#client-authentication), [set the password as `NULL`](#prevent-a-role-from-using-password-authentication).
-`VALID UNTIL timestamp`/`VALID UNTIL NULL` | The date and time (in the [`timestamp`](timestamp.html) format) after which the password is not valid.
+`SQLLOGIN`/`SQLNOLOGIN` | Allow or disallow a role to log in using the SQL CLI with one of the [client authentication methods](authentication.html#client-authentication). Setting the role option to `SQLNOLOGIN` prevents the role from logging in using the SQL CLI with any authentication method. Logins to the DB Console are permitted with `NOSQLLOGIN` set.
+`VALID UNTIL` |  The date and time (in the [`timestamp`](timestamp.html) format) after which the [password](#parameters) is not valid.
+`VIEWACTIVITY`/`NOVIEWACTIVITY` | Allow or disallow a role to see other roles' [queries](show-statements.html) and [sessions](show-sessions.html) using `SHOW STATEMENTS`, `SHOW SESSIONS`, and the [Statements](ui-statements-page.html) and [Transactions](ui-transactions-page.html)  pages in the DB Console. Without this role option, the `SHOW` commands only show the role's own data and the DB Console pages are unavailable. <br><br>By default, the role option is set to `NOVIEWACTIVITY` for all non-admin roles.
+`VIEWACTIVITYREDACTED`/`NOVIEWACTIVITYREDACTED` | Allow or disallow a role to see other roles' queries and sessions using `SHOW STATEMENTS`, `SHOW SESSIONS`, and the Statements and Transactions pages in the DB Console, and the usage of [statements diagnostics bundle](ui-statements-page.html#diagnostics), which can contain PII information. Without this role option, the `SHOW` commands only show the role's own data and the DB Console pages are unavailable. It is possible to have both `VIEWACTIVITY` and `VIEWACTIVITYREDACTED`, and `VIEWACTIVITYREDACTED` takes precedence on restrictions. <br><br>By default, the role option is set to `NOVIEWACTIVITYREDACTED` for all non-admin roles.
```

Comment:
This is great! Can we clarify that the prevention of using the statements diagnostics bundle is only in the console? The user can still run EXPLAIN ANALYZE debug via SQL CLI. 

"and the usage of statements diagnostics bundle in the DB Console"

Reply:
Another thing is that we are using both SQLNOLOGIN and NOSQLLOGIN in the first point. Should use the available option uniformly. 

Also, maybe worth adding a line or two on how that works with LOGIN / NOLOGIN. @thtruo - comments on that?